kuma/users/views.py

```diff
@@ -405,7 +406,11 @@ def get_context_data(self, **kwargs):
                                               .filter(reduced_or_query))
         else:
             matching_accounts = SocialAccount.objects.none()
+
+        registration_disabled = waffle.flag_is_active(self.request,
```

Comment:
I don't think this is even necessary, right?  We can simply do the waffle check within the template?